dev.sh

```diff
@@ -7,6 +7,11 @@ echo "Creating dev environment in ./venv..."
 python3.7 -m venv venv
 . venv/bin/activate
 pip3.7 install -U pip setuptools
+
+# https://github.com/qtile/qtile/issues/994
+echo "Installing xcffib then cairocffi..."
+pip3.7 install --no-cache-dir 'xcffib >= 0.5.0' && pip3.7 install --no-cache-dir 'cairocffi >= 0.9.0'
+
```

Comment:
Either add another `echo` here to say that we're now installing other required packages, or remove the previous one.